components/molecules/selector/Selector.tsx

```
@@ -89,38 +92,62 @@ export const Selector = <T extends Option>({
+        >
           {options.map(option => (
             <li
               key={option.value}
               onClick={() => handleSelect(option)}
-              className="px-3 py-2 text-sm text-text-primary cursor-pointer hover:bg-bg-sub transition-colors"
+              className="flex items-center py-1 pr-2 text-sm text-text-primary cursor-pointer hover:bg-bg-sub transition-colors"
+              role="option"
+              aria-selected={selected?.value === option.value}
             >
-              {option.label}
+              <div
```

Biome (2.3.13)

[error] The HTML element with the interactive role "option" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "option".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)